bin/.travis/prepare_ezpublish.sh

```diff
@@ -8,9 +8,21 @@ echo "> Setup github auth key to not reach api limit"
 echo "> Install dependencies through composer"
 composer install --dev --prefer-dist
 
+echo "> Copy parameters.yml"
+cp bin/.travis/parameters.yml ezpublish/config/
```

Comment:
Shouldn't this ideally be done before composer install? or did it fail for some reason?